MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,781 @@
+---
+layout: post
+title: Working with DataForm in .NET MAUI DataForm control | Syncfusion
+description: Learn about the working with DataForm in Syncfusion .NET MAUI DataForm(SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+# Working with DataForm in Maui DataForm (SfDataForm)
```

Comment:
# Change the data editor settings (SfDataForm)

Reply:
title changed